src/Transition.js

```diff
@@ -394,7 +394,11 @@ Transition.propTypes = {
    *     (see
    *     [test/CSSTransition-test.js](https://github.com/reactjs/react-transition-group/blob/13435f897b3ab71f6e19d724f145596f5910581c/test/CSSTransition-test.js#L362-L437)).
    */
-  nodeRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  nodeRef: PropTypes.shape({ 
+    current: typeof Element === 'undefined' 
+      ? PropTypes.any
+      : PropTypes.instanceOf(Element) 
```

Comment:
Could we not rely on "Element" for this prop-type? My concern is equivalent to https://github.com/merceyz/typescript-to-proptypes/issues/3.

Reply:
The problem can be noticed on https://github.com/mui-org/material-ui/issues/21858#issuecomment-661173090.